packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts

```diff
@@ -432,7 +432,7 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
     // Some environments, like CircleCI, report a large number of CPUs but trying to use them
     // Will cause `Error: Call retries were exceeded` errors.
     // https://github.com/webpack-contrib/terser-webpack-plugin/issues/143
-    const maxCpus = Math.min(cpus().length, 7);
+    const maxCpus = (buildOptions.parallel === undefined) ? Math.min(cpus().length, 7) : buildOptions.parallel;
```

Comment:
Could you please use `Math.max(1, Math.min(cpus().length, 7) - 1)` to avoid freezing when using all CPU?

Before [commit](https://github.com/angular/angular-cli/commit/fd133138e2f6e1248c3ce5e523adb4f378861179 ) here was "true". According to [docs](https://webpack.js.org/plugins/terser-webpack-plugin/#parallel) default is `os.cpus().length - 1`. 

Thanks you!

Reply:
Updated